axelor-account/src/main/java/com/axelor/apps/account/exception/AccountExceptionMessage.java

```diff
@@ -394,6 +394,8 @@ private AccountExceptionMessage() {}
       "The currency is missing on the account move %s" /*)*/;
   public static final String MOVE_13 = /*$$(*/
       "The journal %s is in the 'journals to close' of the period %s, please remove it." /*)*/;
+  public static final String MOVE_15 = /*$$(*/
+      "There is no analytic line on the move line %s." /*)*/;
```

Comment:
Remove this, I thought this check already existed but if it's not the case, we should not add it.